configs/scitokens_credmon.conf

```diff
@@ -0,0 +1,32 @@
+
+# Define the port we will listen to for SciToken / OAuth requests.
+#
+Listen 443 https
+
+# Decide what is a reasonable set of SSL ciphers!  Here's a reasonable recommendation.
+# Instead of blindly accepting this, go to https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=apache-2.4.0&openssl=1.0.1e&hsts=yes&profile=modern
+# and create your own.
+#SSLProtocol             all -SSLv3 -TLSv1 -TLSv1.1
+#SSLCipherSuite          ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
+#SSLHonorCipherOrder     on
+
+<VirtualHost _default_:8443>
+
+  SSLEngine on
+
+  WSGIScriptAlias / /var/www/wsgi-scripts/scitokens-credmon.wsgi
+
+  # A modest number of processes and threads; production applications may want to increase this.
+  WSGIDaemonProcess     CondorCredmon user=condor group=condor processes=2 threads=25
+  WSGIProcessGroup      CondorCredmon
+  WSGIScriptAlias       / /var/www/cgi-bin/wsgi/scitokens-credmon/scitokens-credmon.wsgi
```

Comment:
Ignorant question: Is `/var/www/cgi-bin/wsgi` a typical location for WSGI scripts?

Reply:
I’m not sure if there’s a great convention - I simply copied from a separate project.

Reply:
I did end up simplifying the directory path in the end, but still failed to find any consistency out there in RPM-land.